runtime/near-vm-runner/src/tests/cache.rs

```diff
@@ -124,13 +124,13 @@ fn test_wasmer2_artifact_output_stability() {
     ];
     let mut got_prepared_hashes = Vec::with_capacity(seeds.len());
     let compiled_hashes = [
-        5254981150840481178,
-        15529260255496677612,
-        407257192602619216,
-        10913823971520273759,
-        17423008210698923502,
-        7011050181604188333,
-        15514788595649734538,
```

Comment:
Hmm I’m curious, do you know why the hashes changed? I wouldn’t expect a rustc upgrade to change the hashes of wasmer binaries

Reply:
`rkyv` is sensitive to `repr(Rust)` layout changes, unless the `strict` feature is enabled. Unfortunately that is something we cannot enable as we rely on being able to serialize tuples, and tuples are `repr(Rust)`. So something about how stuff is laid out there has changed.